samples/image-classifier-augmented-manifest-car-damage-detector.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: sagemaker.aws.amazon.com/v1
+kind: TrainingJob
+metadata:
+  name: car-damage-detector
+spec:
+  roleArn: arn:aws:iam::123456789098765:role/service-role/AmazonSageMaker-ExecutionRole
```

Comment:
nit: For consistency with our other samples:
`arn:aws:iam::123456789012:role/service-role/AmazonSageMaker-ExecutionRole`